SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  SKYChatConversationViewController.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
```

Comment:
I don't know too, I will update it to 2017 when I create new files, but let just keep it the current way first, later we will update the existing files in a new PR. 

https://github.com/SkygearIO/chat-SDK-iOS/issues/19